Mathlib/Analysis/InnerProductSpace/PiL2.lean

```diff
@@ -677,10 +672,9 @@ def Complex.isometryOfOrthonormal (v : OrthonormalBasis (Fin 2) ℝ F) : ℂ ≃
 @[simp]
 theorem Complex.map_isometryOfOrthonormal (v : OrthonormalBasis (Fin 2) ℝ F) (f : F ≃ₗᵢ[ℝ] F') :
     Complex.isometryOfOrthonormal (v.map f) = (Complex.isometryOfOrthonormal v).trans f := by
-  simp only [isometryOfOrthonormal, OrthonormalBasis.map, LinearIsometryEquiv.symm_trans,
+  simp [isometryOfOrthonormal, OrthonormalBasis.map, LinearIsometryEquiv.symm_trans,
```

Comment:
This could be a `simp only`, but on the other hand the current form is nice since it would warn us when we add a new `simp` lemma that allows us to remove the `rfl`.
```suggestion
  simp only [isometryOfOrthonormal, OrthonormalBasis.map, LinearIsometryEquiv.symm_trans,
```